src/signals/incident/definitions/wizard-step-5-samenvatting.js

```diff
@@ -57,6 +60,56 @@ export const summary = controls =>
     {}
   );
 
+const expandQuestions = memoize(
```

Comment:
Why do we need a memoize here?

Reply:
So it won't break the `react-reactive-form`.